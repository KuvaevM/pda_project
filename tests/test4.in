start=[S]
[S] = `(` [S] `)` [S] | EPS;