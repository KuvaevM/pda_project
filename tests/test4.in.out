This grammar is not in Greibach normal form.
